wpcsite/settings.py

```diff
@@ -69,6 +70,14 @@ def rel_path(*subpaths):
     return path.join(BASE_DIR, *subpaths)
 
 
+def env_to_bool(environ_key, default):
```

Comment:
This rubs me as wrong, kind of wasteful boilerplate. On other projects I've started to successfully use [python-decouple](https://pypi.python.org/pypi/python-decouple) to do such things. Maybe that'd be a nice cleanup task at some point? Anyways, not a blocker.